recipes/r-prabclus/2.2_6/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+{% set package = 'prabclus' %}
+{% set version = '2.2-6' %}
+
+package:
+  name: "r-{{ package }}"
+  version: "{{ version | replace('-', '_') }}"
+
+source:
+  fn: "{{ package }}_{{ version | replace('-', '_') }}.tar.gz"
+  url:
+    - "https://cran.r-project.org/src/contrib/{{ package }}_{{ version }}.tar.gz"
+    - "http://cran.r-project.org/src/contrib/Archive/{{ package }}/{{ package }}_{{ version }}.tar.gz"
+  sha256: 41792980e40ba18204fab92d85120dcd468860e2204e52fb42636c6f7aee5a62
+
+build:
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base >=2.10
```

Comment:
These min versions for r-base are in general not necessary (the same holds e.g. for python). As far as I know, they might even keep conda-build from correctly pinning r-base. Can you please remove them?